README.md

```diff
@@ -1,5 +1,7 @@
 # Snowflake Guides
 
+[![Builder](https://github.com/bculberson/sfguides/actions/workflows/build.yml/badge.svg)](https://github.com/bculberson/sfguides/actions/workflows/build.yml)
```

Comment:
We should update the url here to point to Snowflake Labs rather than personal.

Reply:
this leaked in, that won't work in this project at all right now. removing